Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/common/ClusterManagerEx.java

```diff
@@ -77,6 +77,11 @@ public static ClusterManagerEx getInstance() {
             synchronized (ClusterManagerEx.class) {
                 if (instance == null) {
                     instance = new ClusterManagerEx();
+
+                    AuthMethodManager.getInstance().addSignOutEventListener(() -> {
+                        // Clean cached clusters
+                        instance.setCachedClusters(instance.additionalClusterDetails);
```

Comment:
Are the linked Esp clusters still work if we sign out and sign in with another Azure account?

Reply:
The linked cluster are all in `additionalClusterDetails`